controller/argument_value_resolver.rst

```diff
@@ -53,17 +53,6 @@ In addition, some components and official bundles provide other value resolvers:
     the controller can be accessed by anonymous users. It requires installing
     the :doc:`Security component </components/security>`.
 
-:class:`Symfony\\Bundle\\SecurityBundle\\SecurityUserValueResolver`
```

Comment:
Wow (unrelated to this PR), this is such an implementation detail that I would even vote to completely remove it from 4.2 docs (and just replace the class name and act like nothing changed, it doesn't matter for a framework user)